apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -446,6 +446,61 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
         );
     };
 
+    /**
+     * A predicate that checks whether a give federated authenticator
+     * is a default authenticator.
+     * @param auth
```

Comment:
```suggestion
     * @param {FederatedAuthenticatorWithMetaInterface} auth - Authenticator.
```

Reply:
Applied this doc comment in commit ba81c1a